Add option to enable content based deduplication via CLI (#47735)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-cli/topic-create.md

| `--supported-codecs`           | Supported data compression methods. Set with a comma.<br/>The default value is `raw`.<br/>Possible values:<ul><li>`RAW`: No compression.</li><li>`ZSTD`: [zstd](https://en.wikipedia.org/wiki/Zstandard) compression.</li><li>`GZIP`: [gzip](https://en.wikipedia.org/wiki/Gzip) compression.</li><li>`LZOP`: [lzop](https://en.wikipedia.org/wiki/Lzop) compression.</li></ul> |
| `--metering-mode`              | The topic pricing method for a serverless database.<br/>Possible values:<ul><li>`request-units`: Based on actual usage.</li><li>`reserved-capacity`: Based on dedicated resources.</li></ul>                                                                                                                                                                                    |

> | `--content-based-deduplication` | Enables content-based message deduplication for the topic. When enabled, messages with identical content are treated as duplicates and only the first message is stored. This is useful for FIFO topics where exact-once semantics are required based on message content rather than explicit deduplication IDs.                                                                |

Reviewer: 1. `exact-once` → `exactly-once` (стандартное выражение, смотри `docs/en/core/concepts/datamodel/topic.md`).
2. `FIFO topics` терминология AWS SQS, а не YDB Topics. Если хочется использовать, надо сделать новый пункт в глоссами

```suggestion
| `--content-based-deduplication` | Enables content-based message deduplication for the topic. When enabled, messages with identical content are treated as duplicates and only the first message is stored.<br/>The default value is `false`.                                                                                                                                                                                                                                                                        |
```

Author: Изменения в документации удалены из ПР - будут в отдельном
